inference.mdx

```diff
@@ -53,12 +53,5 @@ print(response.choices[0].message.content)
 <Warning>
 **Important**
```

Comment:
```suggestion
<Info>
```
I'm not convinced this needs to be in a callout at all tbh, but it is not a warning.